crates/core/plugin_sm/src/plugin.rs

```diff
@@ -436,10 +438,29 @@ impl Plugin for ExternalPluginCommand {
         let command = self.command(LIST, None)?;
         let output = self.execute(command, logger).await?;
         if output.status.success() {
-            Ok(deserialize_module_info(
-                self.name.clone(),
-                output.stdout.as_slice(),
-            )?)
+            let config = get_tedge_config().unwrap();
```

Comment:
I don't think this `unwrap()` is safe because if `/etc/tedge/tedge.toml` doesn't exist or is malformed, it should panic.